### PR TITLE
Sonic boom players for all players, not just localplayer

### PR DESCRIPTION
**Pull request by lalalune**
This fixes a bug where all players are creating a sonic boom app for local player when they join the world. Instead, the player is referenced into the sonic boom app, the result is sonic boom works in multiplayer.

### CODE REVIEW

**Comment by alisaad673** [comment]
Sonic boom is already working for local player though, but  I am unable to test it in multiplayer as multiplayer rooms aren't working for me at all.

image

**Comment by lalalune** [comment]
@alisaad673 this was extracted from multiplayer PR, so while it definitely works, it is dependent on multiplayer to QA -- we can wait until those PRs are merged in

File: character-fx.js
Comment (lalalune) [inline character-fx.js:239]: make player id a component, if its null player then is local player, otherwise look itup

File: metaverse_modules/sonic-boom/index.js
Comment (avaer) [inline metaverse_modules/sonic-boom/index.js:10]: If the player is being passed in, it should be a component (`playerId`). If it's `null` it can mean local, otherwise we can look it up as a remote player.

In general the interface to programming apps should be components only.

**Review by avaer — CHANGES_REQUESTED** [review]
Suggest cleaning up the configuration of the app to go through `components`.

**Pull request merged**
